runtime/vm/jfr.cpp

```diff
@@ -197,14 +285,13 @@ jfrClassesUnload(J9HookInterface **hook, UDATA eventNum, void *eventData, void *
 	j9tty_printf(PORTLIB, "\n!!! class unload %p\n", currentThread);
 #endif /* defined(DEBUG) */
 
-	flushAllThreadBuffers(currentThread);
+	flushAllThreadBuffers(currentThread, false);
```

Comment:
Is this called after the class segments are free'd or before?

Reply:
This hook is called before any unloading starts. If we need to support class unload events, we'll need to add a post hook (but before the classes actually become invalid).